source/aws.rst

```diff
@@ -0,0 +1,141 @@
+.. _aws:
+
+Amazon Web Services (AWS)
+=========================
+
+We often deploy to Amazon Web Services. Here are some tips on getting started.
+
+AWS Accounts vs. IAM Users
+--------------------------
+
+With AWS, we need to be careful to distinguish *AWS accounts* from *IAM users*.
+
+*AWS Accounts* are connected to billing, and to visibility and access to resources.
+Caktus has an AWS account. Some of our clients have their own AWS accounts that
+we might use when working for those clients, so that they get billed
+directly for the resources used instead of us.
+
+*IAM Users* are created within accounts to let administrators
+"log into" those accounts and access those resources. IAM Users within
+one account are unrelated to IAM users in any other account. If I have
+IAM users named "johndoe" in two different AWS accounts, they are completely
+separate and different, having nothing to do with each other.
+
+For you to do things in the Caktus AWS account, someone
+will have to create an *IAM user* for you within that AWS account, and give it the
+privileges for what you need to do.
+
+Root users vs. IAM users
+------------------------
+
+Just to confuse things, *AWS accounts* each have a *root user*, which is not
+an IAM user within that AWS account, but a user in the global Amazon system of
+users - the same set of users that can buy things on Amazon's shopping
+sites, in fact. To create a new AWS account, you have to log in as one of
+this type of user.
+
+Logging in
+----------
+
+To login using a *root user*, there's a global url:
+`https://www.amazon.com/ap/signin <https://www.amazon.com/ap/signin>`_.
```

Comment:
This URL now 404s... Unfortunately, I don't there is a direct link that is guaranteed to work. The AWS docs suggest going to the main console login and then clicking on the 'root url' link if you're not at the right place: https://docs.aws.amazon.com/IAM/latest/UserGuide/console.html#BrowserDefault

:sigh:

Reply:
Thanks for spotting that. I rewrote that part to just go to the main signin and use the 'root url' link.

I also added a section about how to get in and out of subaccounts, which I don't think we'd been doing yet when I wrote this. Please see if that makes sense.  Also, do you think I need to say anything about creating subaccounts or giving people access to them?  (I'd have to find out myself.)

Reply:
Nice! Thanks for adding that info about subaccounts. I've also never created a subaccount, so not sure it needs to be documented here.  :shipit: